kuma/payments/context_processors.py

```diff
@@ -1,12 +1,13 @@
 from .forms import ContributionForm
-from .utils import enabled, popup_enabled
+from .utils import enabled, popup_enabled, recurring_payment_enabled
 
 
 def global_contribution_form(request):
     """Adds contribution form to the context."""
     if enabled(request):
         return {
             'contribution_enabled': True,
+            'contribution_recurring_payment_enabled': recurring_payment_enabled(request),
```

Comment:
This should be ``recurring_payment_enabled`` or ``payment_recurring_enabled`` to avoid "contribution" for new code.